ImperatorToCK3/Source/Imperator/Genes/AccessoryGene.h

```diff
@@ -0,0 +1,28 @@
+#ifndef IMPERATOR_ACCESSORY_GENE_H
+#define IMPERATOR_ACCESSORY_GENE_H
+
+#include "AccessoryGeneTemplate.h"
+#include "Parser.h"
+
+namespace ImperatorWorld
+{
+	class AccessoryGene : commonItems::parser
+	{
+	public:
+		AccessoryGene() = default;
+		explicit AccessoryGene(std::istream& theStream);
+
+		[[nodiscard]] const auto& getIndex() const { return index; }
```

Comment:
No need for const auto& here. When returning primitives returning them is actually cheaper as a copy than as a const reference. Simple auto will do.